sunpy/net/vso/vso.py

```diff
@@ -9,47 +9,41 @@
 This module provides a wrapper around the VSO API.
 """
 
-import re
 import os
+import re
 import sys
-import logging
-import warnings
 import socket
+import warnings
 import itertools
-
 from datetime import datetime, timedelta
 from functools import partial
 from collections import defaultdict
-from suds import client, TypeNotFound
 from urllib.error import URLError, HTTPError
 from urllib.request import urlopen
 
+import zeep
+from zeep.helpers import serialize_object
+
 import astropy.units as u
 from astropy.table import QTable as Table
 
 from sunpy import config
 from sunpy.net import download
-from sunpy.net.base_client import BaseClient
-from sunpy.net.proxyfix import WellBehavedHttpTransport
-from sunpy.util.net import get_filename, slugify
-from sunpy.net.attr import and_
-from sunpy.net.vso import attrs
-from sunpy.net.vso.attrs import walker, TIMEFORMAT
-from sunpy.util import replacement_filename
 from sunpy.time import parse_time
+from sunpy.util import replacement_filename
+from sunpy.net.vso import attrs
+from sunpy.net.attr import and_
+from sunpy.util.net import slugify, get_filename
+from sunpy.net.vso.attrs import TIMEFORMAT, walker
+from sunpy.net.base_client import BaseClient
 
 TIME_FORMAT = config.get("general", "time_format")
 
 DEFAULT_URL_PORT = [{'url': 'http://docs.virtualsolar.org/WSDL/VSOi_rpc_literal.wsdl',
-                     'port': 'nsoVSOi', 'transport': WellBehavedHttpTransport}]
+                     'port': 'nsoVSOi'}]
```

Comment:
I suppose we can add the SDAC wsdl when we get it working w/ zeep.  That should require just a few more lines.  I think it makes more sense just to fail all the way over to the other wsdl at SDAC instead of failing over to the SDAC port on the NSO wsdl.  That makes us more-robust against a cataclysmic failure at NSO.

Reply:
yeah, I think we have all the machinery inplace to fail over to whatever wsdl is online. I don't think we test if the things the wsdl uses are online though.